sa/storage-authority.go

```diff
@@ -579,6 +623,14 @@ func (ssa *SQLStorageAuthority) AddCertificate(certDER []byte, regID int64) (dig
 		RevokedReason:      0,
 		LockCol:            0,
 	}
+	issuedNames := make([]issuedNameModel, len(parsedCertificate.DNSNames))
+	for i, name := range parsedCertificate.DNSNames {
+		issuedNames[i] = issuedNameModel{
+			ReversedName: core.ReverseName(name),
+			Serial:       serial,
+			Issued:       parsedCertificate.NotBefore,
```

Comment:
s/Issued:/NotBefore:/ here and elsewere. (You were mislead by a bug in the SA and then figured it out and made #889 but didn't change this here.)